packages/story-editor/src/components/panels/design/animation/effectPanel.js

```diff
@@ -126,7 +126,7 @@ function EffectPanel({
         effectConfig={config}
         field={field}
         onChange={(value, submitArg) =>
-          handleInputChange({ [field]: value }, submitArg)
+          handleInputChange({ [field]: Math.max(value, 0) }, submitArg)
```

Comment:
I think this can tackle what you've suggested: https://github.com/GoogleForCreators/web-stories-wp/issues/11530

I think it's worth investigating further. This fix does fix the issue at hand, but we're coming into a rabbit hole sort of situation as we dig further, so I think for code quality sake, it warrants investigation when we can prioritize doing so.